### ISSUE TITLE
`mkdir` fails due to dropping privileges

### ISSUE THREAD

**Issue by haampie** [fabiensanglard/st#5]
```
$ sudo ./st mkdir /root/.test
EXEC: [ mkdir /root/.test]
mkdir: cannot create directory ‘/root/.test’: Permission denied
```

Maybe this should be a setuid executable so you don't have to run as `sudo`?

**Comment by janimo** [comment]
setuid and check for geteuid() should work

 int main(int argc, char **argv) {
-    if (getuid() != 0) {
+    if (geteuid() != 0) {
         fprintf(stderr,"Only root can startTimeMs/stop the fork connector\n");
         return 0;
     }

**Comment by fabiensanglard** [comment]
Fixed in 3f58f815d403c9f4462a062d0cd836fa2d9c1dc7

**Comment by cwarden** [comment]
> Fixed in [3f58f81](https://github.com/fabiensanglard/st/commit/3f58f815d403c9f4462a062d0cd836fa2d9c1dc7)

Should be a4744cd37a88c1c2f32de2b7fbf77f773d7fd3ed.  Thanks!
